spec.html

```diff
@@ -38322,6 +38322,14 @@ <h1>`Promise.any` Reject Element Functions</h1>
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="sec-promise.anysettled">
+        <h1>Promise.anySettled ( _iterable_ )</h1>
+        <emu-note>
+          <p>The *"anySettled"* property is an alias for the `Promise.race` and was added for consistency in the naming of promise combinators.</p>
+        </emu-note>
+        <p>The initial value of the *"anySettled"* property is the same function object as the initial value of the `Promise.race` property, but the value of the *"name"* property of this function is *"anySettled"*.</p>
```

Comment:
this isn't possible. the options remain:
1. same function object, both named "race"
1. same function object, both named "anySettled" (may not be web compat)
1. different function objects, each with its own name.